### PR TITLE
[MIG] 13.0 Redo of account_financial_report

### PR DESCRIPTION
**Pull request by thomaspaulb**
Excuse the hack and slash approach; I needed a solution to https://github.com/OCA/account-financial-reporting/issues/690 quick and so I redid the migration, keeping the old approach of showing the HTML as a view so that it can be clicked.

### CODE REVIEW

File: account_financial_report/static/src/js/account_financial_report_backend.js [hunk 1]
```diff
@@ -4,17 +4,22 @@ odoo.define("account_financial_report.account_financial_report_backend", functio
...
     var core = require("web.core");
-    var Widget = require("web.Widget");
-    var ControlPanelMixin = require("web.ControlPanelMixin");
+    var AbstractAction = require('web.AbstractAction');
     var ReportWidget = require("account_financial_report.account_financial_report_widget");
 
-    var report_backend = Widget.extend(ControlPanelMixin, {
+    var report_backend = AbstractAction.extend({
+        // TODO: why isnt the controlpanel shown?
+        hasControlPanel: true,
```
Comment (thomaspaulb) [inline account_financial_report/static/src/js/account_financial_report_backend.js:12]: @NL66278 Despite all of my attempts here, the control panel is still not shown. Do you know what is going wrong?

File: account_financial_report/report/templates/trial_balance.xml [hunk 2]
```diff
@@ -782,220 +753,238 @@
...
-                    <!--## amount_total_due_currency-->
-                    <div class="act_as_cell" style="width: 9%;" />
-                    <!--## amount_residual_currency-->
-                    <div class="act_as_cell" style="width: 9%;" />
+                    <t t-if="account.currency_id.id">
+                        <!--## currency_name-->
+                        <div class="act_as_cell" style="width: 4.43%;">
+                            <t
+                                t-set="domain"
+                                t-value="[('account_id', '=', account.account_id.id)]"
```
Comment (thomaspaulb) [inline account_financial_report/report/templates/trial_balance.xml:1106]: These domains should take into account the state, based on the value of 'only_posted_moves'.

File: account_financial_report/static/src/js/account_financial_report_backend.js (hunk 1 above)
Comment (NL66278) [reply account_financial_report/static/src/js/account_financial_report_backend.js:12]: @thomaspaulb I have no idea I do have only an extremely limited experience with javascript in Odoo...

**Comment by gjotten** [comment]
For context, @thomaspaulb had to do this quickly and maybe a bit drastically in order to fix #690

**Pull request closed without merging**